### PR TITLE
Link to new guides

### PR DESCRIPTION
**Pull request by stevhliu**
This PR adds links to the new guides in the Hub docs sidebar.

### CODE REVIEW

File: docs/hub/_sections.yml [hunk 1]
```diff
@@ -19,8 +19,17 @@
 - local: spaces
   title: Spaces documentation (beta)
 
-- local: adding-a-library
-  title: Integrating a library to the Hub
+- local: tutorial-add-library
```
Comment (osanseviero) [inline docs/hub/_sections.yml:22]: I think we should replace `adding-a-library` entirely but keeping the same name.

Comment (osanseviero) [reply docs/hub/_sections.yml:22]: No strong opinion so if you prefer to have the tutorial prefix it's fine

**Comment by osanseviero** [comment]
Thanks a lot!!

File: docs/hub/_sections.yml [hunk 2]
```diff
@@ -20,7 +20,16 @@
...
+  title: Integrate a library with the Hub
+
+- local: how-to-downstream
+  title: How to download files from the Hub
+
+- local: how-to-upstream
+  title: How to create repositories and upload files to the Hub
+
+- local: how-to-inference
+  title: How to set up Inference API with your library
```
Comment (julien-c) [inline docs/hub/_sections.yml:32]: to me this sounds a bit confusing vs. how to actually spawn an inference API if you're a library author

I'd suggest changing to something like `How to call the Inference API from Python code`

WDYT?

**Pull request merged**